src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+class Rectangle extends Figure {
+    private double width;
+    private double height;
+
+    public Rectangle(Color color, double width,double height) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. Create separate variables and pass them on for better code readability.